code/modules/robotics/bot/duckbot.dm

```diff
@@ -87,6 +87,19 @@
 	src.botcard = new /obj/item/card/id(src)
 	src.botcard.access = get_access(src.access_lookup)
 
+/obj/machinery/bot/duckbot/navigate_to(atom/the_target, var/move_delay = 10, var/adjacent = 0, max_dist=60)
+	var/target_turf = get_pathable_turf(the_target)
+	if((get_dist(the_target, src) == 0))
```

Comment:
What's `var/adjacent` for again? If it's a boolean for whether or not it's an adjacent movement, you could just check for it in the original proc.

Reply:
It appears to be so that a bot stands next to the target tile, not on it